db/db_utils.go

```diff
@@ -131,13 +148,27 @@ func EnsureIndex(collection string, index mongo.IndexModel) error {
 	return errors.WithStack(err)
 }
 
-// DropIndex takes in a collection and a slice of keys and drops those indexes
-func DropAllIndexes(collection string) error {
+// DropAllIndexes drops all indexes in the specified collections, returning an
```

Comment:
is there ever a case where we'd use this instead of just dropping the collections?